cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
can we get rid of this file?

Reply:
Going to keep this one for now, in case we add any special configurations!